Sources/Features/Repository/Model/BacktraceReport.swift

```diff
@@ -1,5 +1,9 @@
 import Foundation
+#if canImport(CrashReporter)
+import CrashReporter
+#elseif canImport(Backtrace_PLCrashReporter)
 import Backtrace_PLCrashReporter
```

Comment:
Is this still here for backwards compatibility if someone builds our lib using Cocoa-Pods?

Reply:
For Cocoa pod its will works well, we maintian the backward compatiblity.

Reply:
As we discussed, we do not want to release a version where we rely on a different versions of plcrashreporter for SwiftPM vs Cocoapods.

Please change the Podfile to use the microsoft official cocoa pod:
https://cocoapods.org/pods/PLCrashReporter